src/app/components/challenge/challengesubmit/challengesubmit.component.html

```diff
@@ -241,67 +258,65 @@
-              </strong>
-            </span>
-          </div>
-        </div>
+      class="col-md-5 col-sm-5 col-xs-12">
+      <div class="ev-card-body exist-team-card ev-remaining-submission
+      remaining-submission-clock">
+      <div class="row row-lr-margin">
+        <span class="color-black">
+          <strong class="fw-regular fs-15">
+            Today's remaining Submissions:
+          </strong>
+        </span>
+        <span class="color-black">
+          <strong class="fw-light fs-15">
+            {{selectedPhaseSubmissions.remainingSubmissions.remaining_submissions_today_count}}
+          </strong>
+        </span>
       </div>
-
-      <div *ngIf="selectedPhaseSubmissions.showClock" class="col-md-5 col-sm-12 col-xs-12">
-        <div class="ev-card-body exist-team-card ev-remaining-submission
-          remaining-submission-clock">
-          <div class="row row-lr-margin">
-            <strong class="fw-regular text-med-black">
-              Message:
-            </strong> {{selectedPhaseSubmissions.clockMessage.message}}
-          </div>
-          <div class="fw-regular">
-            Next set of submissions will be available after:
-          </div>
-          <div>
-            <span *ngIf="days!=0">{{days}} Days, </span>{{hours}}
-            hours : {{minutes}} minutes : {{seconds}} seconds
-          </div>
-        </div>
+      <div class="row row-lr-margin">
+        <span class="color-black">
+          <strong class="fw-regular fs-15">
+            Monthly remaining submissions:
+          </strong>
+        </span>
+        <span class="color-black">
+          <strong class="fw-light fs-15">
+            {{selectedPhaseSubmissions.remainingSubmissions.remaining_submissions_this_month_count}}
+          </strong>
+        </span>
+      </div>
+      <div class="row row-lr-margin">
+        <span class="color-black">
+          <strong class="fw-regular fs-15">
+            Total remaining submissions:
+          </strong>
+        </span>
+        <span class="color-black">
+          <strong class="fw-light fs-15">
+            {{selectedPhaseSubmissions.remainingSubmissions.remaining_submissions_count}}
+          </strong>
+        </span>
       </div>
     </div>
   </div>
+  <div *ngIf="selectedPhaseSubmissions.showClock" class="col-md-5 col-sm-12 col-xs-12">
+    <div class="ev-card-body exist-team-card ev-remaining-submission 
+    remaining-submission-clock">
+      <div class="row row-lr-margin">
+        <strong class="fw-regular text-med-black">
+          Message:
+        </strong> {{selectedPhaseSubmissions.clockMessage.message}}
+      </div>
+      <div class="fw-regular">
+        Next set of submissions will be available after:
+      </div>
+      <div>
+        <span *ngIf="days!=0">{{days}} Days, </span>{{hours}}
+        hours : {{minutes}} minutes : {{seconds}} seconds
+      </div>
+    </div>
+  </div>
+</div>
+</div>
```

Comment:
Still its unindented. Each child element should be 1 tab in space from its parent element.

Reply:
so i have to give one space instead of two?

Reply:
done sir